### DISCUSSION TITLE
0.7.7

### DISCUSSION THREAD

**Discussion by SchrodingersGat** [inventree/InvenTree#3407] [Announcements]
### Bug Fixes
* Make sure PIP is upgraded to latest version by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/3402
### Other Changes
* Increment version number to 0.7.7 by @SchrodingersGat in https://github.com/inventree/InvenTree/pull/3403

**Full Changelog**: https://github.com/inventree/InvenTree/compare/0.7.6...0.7.7

<hr /><em>This discussion was created from the release <a href='https://github.com/inventree/InvenTree/releases/tag/0.7.7'>0.7.7</a>.</em>

**Comment by miggland** [comment]
@SchrodingersGat just wondering what happened to some other commits and PRs which were merged around the same - they are not included in this new release. Any reason for that?

**Reply by SchrodingersGat** [reply]
Any in particular you thought should be included? This release was to patch a particular docker bug

**Reply by miggland** [reply]
The [milestone lists 6 issues](https://github.com/inventree/InvenTree/milestone/26?closed=1), but only two of those are reference in the release notes..

**Reply by SchrodingersGat** [reply]
You're right, some of those were incorrectly tagged. Thanks for picking it up.

**Reply by matmair** [reply]
@miggland we are automating the boring stuff - including release notes ;-) So the tags are kind of important.

**Reply by miggland** [reply]
@matmair OK. But which tag was even wrong? I still can't see a difference, except now that the missing PRs have been moved to another milestone.

**Reply by matmair** [reply]
@miggland the tags and the milestones are used to write the release notes automated so they need to be set before merging correctly.
